docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Management and Monitoring REST API/definitions.adoc

```diff
@@ -1,5 +1,31 @@
 = Definitions
```

Comment:
The file name should have a capital 'D' - its not found in the nav.adoc because of this

Reply:
Ready to merge after this change